ops/cpis/softlayer/vars.tmpl

```diff
@@ -1,4 +1,5 @@
 director_name: # bosh
+stemcell: bosh-softlayer-xen-ubuntu-trusty-go_agent
```

Comment:
do we want this to be something changable? otherwise we could also just create and ops-file in each cpi directory to add cpi and stemcell to credhub importer